app/dashboard/sync/celo.py

```diff
@@ -10,7 +11,11 @@ def find_txn_on_celo_explorer(fulfillment, network='mainnet'):
         return None
 
     funderAddress = fulfillment.bounty.bounty_owner_address
-    amount = fulfillment.payout_amount
+
+    token = Token.objects.filter(symbol=token_name).first()
```

Comment:
maybe worth handling the falsy `token` case?

Reply:
Will add in defaults just as backup